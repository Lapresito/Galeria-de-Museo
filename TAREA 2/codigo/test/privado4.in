crearGrupoABB
liberarGrupoABB
crearGrupoABB
crearVisitante 100 Brunilda Gomez 35
agregarAGrupoABB
crearVisitante 200 Casimira Ruiz 23
agregarAGrupoABB
crearVisitante 160 Zenobia Garcia 41
agregarAGrupoABB
crearVisitante 240 Brunilda Perez 17
agregarAGrupoABB
crearVisitante 300 Zenobia Gonzalez 23
agregarAGrupoABB
crearVisitante 310 Rufino Fernandez 25
agregarAGrupoABB
crearVisitante 320 Casimira Perez 17
agregarAGrupoABB
crearVisitante 400 Ubaldo Garcia 22
agregarAGrupoABB
crearVisitante 80 Esmeralda Fernandez 24
agregarAGrupoABB
crearVisitante 40 Filemon Lopez 43
agregarAGrupoABB
crearVisitante 60 Casimira Martinez 19
agregarAGrupoABB
crearVisitante 70 Rufino Martinez 26
agregarAGrupoABB
alturaGrupoABB
existeEnGrupoABB 60
existeEnGrupoABB 310
existeEnGrupoABB 20
existeEnGrupoABB 180
maxIdGrupoABB
edadPromedioGrupoABB
removerDeGrupoABB 300
removerDeGrupoABB 80
removerDeGrupoABB 310
removerDeGrupoABB 240
alturaGrupoABB
existeEnGrupoABB 60
existeEnGrupoABB 310
maxIdGrupoABB
edadPromedioGrupoABB
liberarGrupoABB
Fin
