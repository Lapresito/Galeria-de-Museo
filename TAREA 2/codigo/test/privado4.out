Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El grupoABB ha sido creado de forma exitosa.
2> Grupo liberado con exito.
3> El grupoABB ha sido creado de forma exitosa.
4> Visitante creadox de forma exitosa.
5> Se ha agregado el visitante al grupo de forma exitosa.
6> Visitante creadox de forma exitosa.
7> Se ha agregado el visitante al grupo de forma exitosa.
8> Visitante creadox de forma exitosa.
9> Se ha agregado el visitante al grupo de forma exitosa.
10> Visitante creadox de forma exitosa.
11> Se ha agregado el visitante al grupo de forma exitosa.
12> Visitante creadox de forma exitosa.
13> Se ha agregado el visitante al grupo de forma exitosa.
14> Visitante creadox de forma exitosa.
15> Se ha agregado el visitante al grupo de forma exitosa.
16> Visitante creadox de forma exitosa.
17> Se ha agregado el visitante al grupo de forma exitosa.
18> Visitante creadox de forma exitosa.
19> Se ha agregado el visitante al grupo de forma exitosa.
20> Visitante creadox de forma exitosa.
21> Se ha agregado el visitante al grupo de forma exitosa.
22> Visitante creadox de forma exitosa.
23> Se ha agregado el visitante al grupo de forma exitosa.
24> Visitante creadox de forma exitosa.
25> Se ha agregado el visitante al grupo de forma exitosa.
26> Visitante creadox de forma exitosa.
27> Se ha agregado el visitante al grupo de forma exitosa.
28> La altura del grupo es 7.
29> El visitante con id 60 pertenece al grupo.
30> El visitante con id 310 pertenece al grupo.
31> El visitante con id 20 NO pertenece al grupo.
32> El visitante con id 180 NO pertenece al grupo.
33> El mayor id en el grupo es 400.
34> La edad promedio del grupo es 26.25.
35> El visitante con id 300 se removio del grupo.
36> El visitante con id 80 se removio del grupo.
37> El visitante con id 310 se removio del grupo.
38> El visitante con id 240 se removio del grupo.
39> La altura del grupo es 4.
40> El visitante con id 60 pertenece al grupo.
41> El visitante con id 310 NO pertenece al grupo.
42> El mayor id en el grupo es 400.
43> La edad promedio del grupo es 28.25.
44> Grupo liberado con exito.
45> Fin.
